src/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,38 @@
+import NextAuth from "next-auth";
+import GoogleProvider from "next-auth/providers/google";
+import CredentialsProvider from "next-auth/providers/credentials"
+
+export const authOptions = {
+  providers: [
+    GoogleProvider({
+        clientId: process.env.GOOGLE_ID as string,
+        clientSecret: process.env.GOOGLE_SECRET as string
```

Comment:
these are .env variable, when you add new ,env vars you should edit the .env.example file and add them there as well, but we don;t have one so you can create one 

this would be in the root dir, the name is .env.example, the content would be

```env
GOOGLE_ID= # you can write a small comment about what is this variable
GOOGLE_SECRET=
```

Reply:
done

Reply:
l already have such a file named **.env.local** on my local in the folder oot. 
![Screen Shot 2023-08-08 at 12 34 39 PM](https://github.com/MohammedAlabd/social-network-nextjs/assets/86113303/b3f54a75-9a66-40e3-9c87-0fd9e9c23def)

If you want me to change the name to **.env.example** it will be a folder that is not secured and will be on GitHub in this repository and all the Okeys will be displayed to anyone, right? I think I miss understood your request